lib/modules/play/widgets/audio_slider.dart

```diff
@@ -27,29 +24,49 @@ class _AudioSliderState extends State<AudioSlider> {
   @override
   Widget build(BuildContext context) {
     final audio = context.watch<AudioController>();
+    final current = audio.key == (widget.playerKey ?? widget.url);
     if (!drag) value = audio.position;
-    if (widget.collapsible && audio.key != (widget.playerKey ?? widget.url)) {
-      return Container(
-        color: Theme.of(context).colorScheme.primary,
-        constraints: const BoxConstraints(minHeight: 3),
-      );
-    }
-    return slideUp(
-      key: Key(audio.key),
-      child: Slider(
-        value: value < audio.duration ? value.toDouble() : 0,
-        max: audio.duration.toDouble(),
-        onChanged: (s) => setState(() {
-          drag = true;
-          value = s.toInt();
-        }),
-        onChangeEnd: (s) async {
-          await audio.seek(s.toInt());
-          setState(() {
-            drag = false;
-          });
-        },
-      ),
+    return Row(
+      children: [
+        IconButton(
+          onPressed: () => setState(
+            () => current
+                ? audio.playing
+                    ? audio.pause()
+                    : audio.resume()
+                : audio.play(widget.url, widget.playerKey),
```

Comment:
I generally don't like arrow functions when the return value is not needed. In this case, it would even throw a runtime error if `pause`/`resume`/`play` return a `Future`.
Also, the ternary operator gets really unreadable really quickly
```suggestion
            () {
              if (current) {
                if (audio.playing) {
                  audio.pause();
                } else {
                  audio.resume();
                }
              } else {
                audio.play(widget.url, widget.playerKey);
              }
            }
```
It might be longer, but more readable